scanner/docker/runtime-entrypoint.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
It seems I left this unused file around in `main`, I've now deleted it, probably a good idea to merge `main` again